docs(fix): Add "Suggest an Edit" link to "Make a Change Using the GitHub Web UI" page (#1828)
spinnaker/spinnaker.github.io: community/contributing/docs/github-changes.md

# Open a pull request on GitHub

> 1. Click the **Suggest an Edit** link in the upper right hand corner of the Spinnaker documentation page. This takes you to the file in GitHub.

Reviewer: ```suggestion
1. Click the **Suggest an Edit** link on the documentation page you want to update. This takes you to the file in GitHub.
```

Reviewer: The reason I suggest this change is that directional descriptions like "upper right" are considered less accessible. Also, if we move the link this'll be broken. But unless we change the text of the link, this will still be findable.

> 1. Select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.

Reviewer: Same thing here. Maybe "pencil icon at the top right" --> "**edit** button" or "**Edit this file** button".

Author: I followed the steps. I didn't see an "Edit this page" link or button anywhere, only the pencil icon. Removed the directional but left pencil icon.

Reviewer: "Edit this file" is the hover text for the icon, so that's what someone using a sceen reader would hear. Of course, that confuses sighted users, who only see the pencil. :) No biggie for now, but I'll try to remember to look up best practices for this.

Reviewer: Something like "Click the **Edit this file** (✏️) button."

Author: where did you get that pencil icon? Unfortunately I can't see the comment source.

Author: Added an image that points out the pencil icon.

1. Make your changes in the GitHub markdown editor.

> 1. Below the editor, fill in the **Propose file change** form.

Reviewer: Also might want to avoid this "Below" directionality.

# Open a pull request (PR) on GitHub

1. Click the **Suggest an Edit** link on the documentation page you want to update. This takes you to the page's source file in GitHub.

> 1. Click the **Edit this file in the fork of your project** pencil icon to edit the file.

Reviewer: Hmm, for me it just says "Edit this file"

![edit_this_file](https://user-images.githubusercontent.com/29312809/80819669-6c3ba600-8ba3-11ea-88ef-b76e76fbda01.png)

Reviewer: Either way, I think it's fine just to say "**Edit this file** icon"

Author: That's bizarre. I'm on a Mac, and the hover help is the same regardless of browser.  I don't see the computer monitor icon in your screenshot either.
![edit-the-file](https://user-images.githubusercontent.com/26799583/80821482-6db69f80-8b9e-11ea-83d0-cfd9ca7d01f7.png)

Author: Changed to "Edit this file"

Reviewer: @aimeeu do you have merge rights in the repo? It's possible that I'm seeing something different because I do. Thus you're seeing what the user will see, and I'm not. "Edit this file..." with the elipsis, seems like the safe choice.

Author: Nope - no merge rights. I'm just a lowly contributor. LOL. 
That explains why you don't see the "in your fork" portion.

  For more information, see [commit message guidelines](https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions).

>   In the **second field**, provide a clear and detailed description of your PR. Do not to leave these fields blank, as it is helpful to reviewers when merging your request to have additional context as to what your pull request is about.

Reviewer: These "first field," "second field" don't work to well either, but that's how it was before this PR, so ok for now. (Heck, I probably wrote that myself 🌵 )

Author: @dorbin A new contributor submitted this PR during Spinnaker Gardening Days. After reading your comments, I followed the instructions, and then reordered. Also ensured field names matched the UI. Moved the PR title checker content to the `Open a pull request` step because GitHub only checks the PR title, not the title of the commit. Added two images for clarity.

Reviewer: 👍
